src/main/java/org/jabref/gui/preferences/PreferencesDialogView.java

```diff
@@ -88,7 +86,7 @@ private void initialize() {
                 .withText(PreferencesTab::getTabName)
                 .install(preferenceTabList);
 
-        viewModel.setValues(); // ToDo: Remove this after conversion of all tabs
+        DefaultTaskExecutor.runInJavaFXThread(viewModel::setValues);
```

Comment:
Could you please add a comment why this is necessary?

Reply:
The FXThread or setValues()?
This is something I forgot to put in a BackgroundTask, @tobiasdiez asked to lazy-load the prefs, I did not have time to look deeper into it.

Reply:
is this a bad idea?
`BackgroundTask.wrap(() -> DefaultTaskExecutor.runInJavaFXThread(() -> viewModel.setValues())).executeWith(Globals.TASK_EXECUTOR);`

Reply:
Oh, I formulated that in a quite misleading way. What I meant is that `tab.setValues` should only be called when `tab` gets the focus - and not when the preference dialog is loaded. I was hoping to improve the loading time of the dialog in that way. But this kind of lazy loading probably needs a bit of refactoring because you don't want to override the users temporary changes when he switches between tabs. You can leave this also for another PR.